Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 5
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	99	discretize_counts
	100

[Fri Jan 29 23:13:01 2021]
rule discretize_counts:
    input: read_counts/AML-01-001_AD.csv, read_counts/AML-01-001_DP.csv
    output: input/AML-01-001_input.csv, input/AML-01-001.loci
    jobid: 1
    wildcards: patient=AML-01-001

[Fri Jan 29 23:13:01 2021]
rule discretize_counts:
    input: read_counts/AML-76-001_AD.csv, read_counts/AML-76-001_DP.csv
    output: input/AML-76-001_input.csv, input/AML-76-001.loci
    jobid: 76
    wildcards: patient=AML-76-001

[Fri Jan 29 23:13:01 2021]
rule discretize_counts:
    input: read_counts/AML-06-001_AD.csv, read_counts/AML-06-001_DP.csv
    output: input/AML-06-001_input.csv, input/AML-06-001.loci
    jobid: 5
    wildcards: patient=AML-06-001


[Fri Jan 29 23:13:01 2021]
rule discretize_counts:
    input: read_counts/AML-28-001_AD.csv, read_counts/AML-28-001_DP.csv
    output: input/AML-28-001_input.csv, input/AML-28-001.loci
    jobid: 30
    wildcards: patient=AML-28-001

[Fri Jan 29 23:13:01 2021]
rule discretize_counts:
    input: read_counts/AML-26-001_AD.csv, read_counts/AML-26-001_DP.csv
    output: input/AML-26-001_input.csv, input/AML-26-001.loci
    jobid: 28
    wildcards: patient=AML-26-001

[Fri Jan 29 23:13:07 2021]
Error in rule discretize_counts:
    jobid: 1
    output: input/AML-01-001_input.csv, input/AML-01-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp2j6e0_dg.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-01-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:07 2021]
rule discretize_counts:
    input: read_counts/AML-31-001_AD.csv, read_counts/AML-31-001_DP.csv
    output: input/AML-31-001_input.csv, input/AML-31-001.loci
    jobid: 33
    wildcards: patient=AML-31-001

[Fri Jan 29 23:13:07 2021]
Error in rule discretize_counts:
    jobid: 30
    output: input/AML-28-001_input.csv, input/AML-28-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7sqg7lyz.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-28-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:07 2021]
rule discretize_counts:
    input: read_counts/AML-66-001_AD.csv, read_counts/AML-66-001_DP.csv
    output: input/AML-66-001_input.csv, input/AML-66-001.loci
    jobid: 66
    wildcards: patient=AML-66-001

[Fri Jan 29 23:13:08 2021]
Error in rule discretize_counts:
    jobid: 5
    output: input/AML-06-001_input.csv, input/AML-06-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpmu3t1k97.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-06-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:08 2021]
rule discretize_counts:
    input: read_counts/AML-77-001_AD.csv, read_counts/AML-77-001_DP.csv
    output: input/AML-77-001_input.csv, input/AML-77-001.loci
    jobid: 77
    wildcards: patient=AML-77-001

[Fri Jan 29 23:13:08 2021]
Error in rule discretize_counts:
    jobid: 28
    output: input/AML-26-001_input.csv, input/AML-26-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp4pandenx.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-26-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:08 2021]
rule discretize_counts:
    input: read_counts/AML-51-001_AD.csv, read_counts/AML-51-001_DP.csv
    output: input/AML-51-001_input.csv, input/AML-51-001.loci
    jobid: 51
    wildcards: patient=AML-51-001

[Fri Jan 29 23:13:08 2021]
Error in rule discretize_counts:
    jobid: 76
    output: input/AML-76-001_input.csv, input/AML-76-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp8i2st78x.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-76-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:08 2021]
rule discretize_counts:
    input: read_counts/AML-74-001_AD.csv, read_counts/AML-74-001_DP.csv
    output: input/AML-74-001_input.csv, input/AML-74-001.loci
    jobid: 74
    wildcards: patient=AML-74-001

[Fri Jan 29 23:13:10 2021]
Error in rule discretize_counts:
    jobid: 51
    output: input/AML-51-001_input.csv, input/AML-51-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp7ni_0654.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:10 2021]
rule discretize_counts:
    input: read_counts/AML-43-001_AD.csv, read_counts/AML-43-001_DP.csv
    output: input/AML-43-001_input.csv, input/AML-43-001.loci
    jobid: 44
    wildcards: patient=AML-43-001

[Fri Jan 29 23:13:12 2021]
Error in rule discretize_counts:
    jobid: 33
    output: input/AML-31-001_input.csv, input/AML-31-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpigzzbk_i.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-31-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:12 2021]
rule discretize_counts:
    input: read_counts/AML-14-001_AD.csv, read_counts/AML-14-001_DP.csv
    output: input/AML-14-001_input.csv, input/AML-14-001.loci
    jobid: 16
    wildcards: patient=AML-14-001

[Fri Jan 29 23:13:13 2021]
Error in rule discretize_counts:
    jobid: 77
    output: input/AML-77-001_input.csv, input/AML-77-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpk0twcy3f.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-77-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:13 2021]
rule discretize_counts:
    input: read_counts/AML-73-001_AD.csv, read_counts/AML-73-001_DP.csv
    output: input/AML-73-001_input.csv, input/AML-73-001.loci
    jobid: 73
    wildcards: patient=AML-73-001

[Fri Jan 29 23:13:13 2021]
Error in rule discretize_counts:
    jobid: 66
    output: input/AML-66-001_input.csv, input/AML-66-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpiksvnzup.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-66-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:13 2021]
rule discretize_counts:
    input: read_counts/AML-32-001_AD.csv, read_counts/AML-32-001_DP.csv
    output: input/AML-32-001_input.csv, input/AML-32-001.loci
    jobid: 34
    wildcards: patient=AML-32-001

[Fri Jan 29 23:13:14 2021]
Error in rule discretize_counts:
    jobid: 74
    output: input/AML-74-001_input.csv, input/AML-74-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpoou5oumi.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-74-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:14 2021]
rule discretize_counts:
    input: read_counts/AML-90-001_AD.csv, read_counts/AML-90-001_DP.csv
    output: input/AML-90-001_input.csv, input/AML-90-001.loci
    jobid: 90
    wildcards: patient=AML-90-001

[Fri Jan 29 23:13:15 2021]
Error in rule discretize_counts:
    jobid: 44
    output: input/AML-43-001_input.csv, input/AML-43-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp63f5nlrn.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-43-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:15 2021]
rule discretize_counts:
    input: read_counts/AML-41-001_AD.csv, read_counts/AML-41-001_DP.csv
    output: input/AML-41-001_input.csv, input/AML-41-001.loci
    jobid: 42
    wildcards: patient=AML-41-001

[Fri Jan 29 23:13:18 2021]
Error in rule discretize_counts:
    jobid: 16
    output: input/AML-14-001_input.csv, input/AML-14-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp4zlil_e5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-14-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:18 2021]
rule discretize_counts:
    input: read_counts/AML-09-001_AD.csv, read_counts/AML-09-001_DP.csv
    output: input/AML-09-001_input.csv, input/AML-09-001.loci
    jobid: 8
    wildcards: patient=AML-09-001

[Fri Jan 29 23:13:18 2021]
Error in rule discretize_counts:
    jobid: 34
    output: input/AML-32-001_input.csv, input/AML-32-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp0clzdfaq.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-32-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:18 2021]
rule discretize_counts:
    input: read_counts/AML-67-001_AD.csv, read_counts/AML-67-001_DP.csv
    output: input/AML-67-001_input.csv, input/AML-67-001.loci
    jobid: 67
    wildcards: patient=AML-67-001

[Fri Jan 29 23:13:19 2021]
Error in rule discretize_counts:
    jobid: 73
    output: input/AML-73-001_input.csv, input/AML-73-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpkgqjl4va.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-73-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:19 2021]
rule discretize_counts:
    input: read_counts/AML-02-001_AD.csv, read_counts/AML-02-001_DP.csv
    output: input/AML-02-001_input.csv, input/AML-02-001.loci
    jobid: 2
    wildcards: patient=AML-02-001

[Fri Jan 29 23:13:19 2021]
Error in rule discretize_counts:
    jobid: 90
    output: input/AML-90-001_input.csv, input/AML-90-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp5izw5i80.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-90-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:19 2021]
rule discretize_counts:
    input: read_counts/AML-83-001_AD.csv, read_counts/AML-83-001_DP.csv
    output: input/AML-83-001_input.csv, input/AML-83-001.loci
    jobid: 83
    wildcards: patient=AML-83-001

[Fri Jan 29 23:13:21 2021]
Error in rule discretize_counts:
    jobid: 42
    output: input/AML-41-001_input.csv, input/AML-41-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmphnfchp0v.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-41-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:21 2021]
rule discretize_counts:
    input: read_counts/AML-13-001_AD.csv, read_counts/AML-13-001_DP.csv
    output: input/AML-13-001_input.csv, input/AML-13-001.loci
    jobid: 15
    wildcards: patient=AML-13-001

[Fri Jan 29 23:13:23 2021]
Error in rule discretize_counts:
    jobid: 8
    output: input/AML-09-001_input.csv, input/AML-09-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp5zuwz1c1.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-09-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:23 2021]
rule discretize_counts:
    input: read_counts/AML-40-001_AD.csv, read_counts/AML-40-001_DP.csv
    output: input/AML-40-001_input.csv, input/AML-40-001.loci
    jobid: 41
    wildcards: patient=AML-40-001

[Fri Jan 29 23:13:24 2021]
Error in rule discretize_counts:
    jobid: 67
    output: input/AML-67-001_input.csv, input/AML-67-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmplqtoj2xi.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-67-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:24 2021]
rule discretize_counts:
    input: read_counts/AML-100-001_AD.csv, read_counts/AML-100-001_DP.csv
    output: input/AML-100-001_input.csv, input/AML-100-001.loci
    jobid: 9
    wildcards: patient=AML-100-001

[Fri Jan 29 23:13:24 2021]
Error in rule discretize_counts:
    jobid: 83
    output: input/AML-83-001_input.csv, input/AML-83-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpb0oou2jj.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-83-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:24 2021]
rule discretize_counts:
    input: read_counts/AML-39-001_AD.csv, read_counts/AML-39-001_DP.csv
    output: input/AML-39-001_input.csv, input/AML-39-001.loci
    jobid: 40
    wildcards: patient=AML-39-001

[Fri Jan 29 23:13:24 2021]
Error in rule discretize_counts:
    jobid: 2
    output: input/AML-02-001_input.csv, input/AML-02-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmphf3auyy2.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-02-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:24 2021]
rule discretize_counts:
    input: read_counts/AML-85-001_AD.csv, read_counts/AML-85-001_DP.csv
    output: input/AML-85-001_input.csv, input/AML-85-001.loci
    jobid: 85
    wildcards: patient=AML-85-001

[Fri Jan 29 23:13:26 2021]
Error in rule discretize_counts:
    jobid: 15
    output: input/AML-13-001_input.csv, input/AML-13-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpkrg895s1.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-13-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:26 2021]
rule discretize_counts:
    input: read_counts/AML-98-001_AD.csv, read_counts/AML-98-001_DP.csv
    output: input/AML-98-001_input.csv, input/AML-98-001.loci
    jobid: 98
    wildcards: patient=AML-98-001

[Fri Jan 29 23:13:28 2021]
Error in rule discretize_counts:
    jobid: 85
    output: input/AML-85-001_input.csv, input/AML-85-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmplkqnhl1_.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-85-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:28 2021]
rule discretize_counts:
    input: read_counts/AML-63-001_AD.csv, read_counts/AML-63-001_DP.csv
    output: input/AML-63-001_input.csv, input/AML-63-001.loci
    jobid: 63
    wildcards: patient=AML-63-001

[Fri Jan 29 23:13:29 2021]
Error in rule discretize_counts:
    jobid: 9
    output: input/AML-100-001_input.csv, input/AML-100-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpifg9jlh9.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-100-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:29 2021]
rule discretize_counts:
    input: read_counts/AML-82-001_AD.csv, read_counts/AML-82-001_DP.csv
    output: input/AML-82-001_input.csv, input/AML-82-001.loci
    jobid: 82
    wildcards: patient=AML-82-001

[Fri Jan 29 23:13:29 2021]
Error in rule discretize_counts:
    jobid: 40
    output: input/AML-39-001_input.csv, input/AML-39-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpjffqmbdw.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-39-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:29 2021]
rule discretize_counts:
    input: read_counts/AML-70-001_AD.csv, read_counts/AML-70-001_DP.csv
    output: input/AML-70-001_input.csv, input/AML-70-001.loci
    jobid: 70
    wildcards: patient=AML-70-001

[Fri Jan 29 23:13:29 2021]
Error in rule discretize_counts:
    jobid: 41
    output: input/AML-40-001_input.csv, input/AML-40-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpzm4fobgf.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-40-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:29 2021]
rule discretize_counts:
    input: read_counts/AML-92-001_AD.csv, read_counts/AML-92-001_DP.csv
    output: input/AML-92-001_input.csv, input/AML-92-001.loci
    jobid: 92
    wildcards: patient=AML-92-001

[Fri Jan 29 23:13:31 2021]
Error in rule discretize_counts:
    jobid: 98
    output: input/AML-98-001_input.csv, input/AML-98-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpf_3qbayp.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-98-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:31 2021]
rule discretize_counts:
    input: read_counts/AML-16-001_AD.csv, read_counts/AML-16-001_DP.csv
    output: input/AML-16-001_input.csv, input/AML-16-001.loci
    jobid: 18
    wildcards: patient=AML-16-001

[Fri Jan 29 23:13:33 2021]
Error in rule discretize_counts:
    jobid: 63
    output: input/AML-63-001_input.csv, input/AML-63-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmplli3b355.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-63-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:33 2021]
rule discretize_counts:
    input: read_counts/AML-20-001_AD.csv, read_counts/AML-20-001_DP.csv
    output: input/AML-20-001_input.csv, input/AML-20-001.loci
    jobid: 22
    wildcards: patient=AML-20-001

[Fri Jan 29 23:13:33 2021]
Error in rule discretize_counts:
    jobid: 70
    output: input/AML-70-001_input.csv, input/AML-70-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpix1y42es.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-70-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:33 2021]
rule discretize_counts:
    input: read_counts/AML-88-001_AD.csv, read_counts/AML-88-001_DP.csv
    output: input/AML-88-001_input.csv, input/AML-88-001.loci
    jobid: 88
    wildcards: patient=AML-88-001

[Fri Jan 29 23:13:33 2021]
Error in rule discretize_counts:
    jobid: 82
    output: input/AML-82-001_input.csv, input/AML-82-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpinohz4g1.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-82-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:33 2021]
rule discretize_counts:
    input: read_counts/AML-50-001_AD.csv, read_counts/AML-50-001_DP.csv
    output: input/AML-50-001_input.csv, input/AML-50-001.loci
    jobid: 50
    wildcards: patient=AML-50-001

[Fri Jan 29 23:13:34 2021]
Error in rule discretize_counts:
    jobid: 92
    output: input/AML-92-001_input.csv, input/AML-92-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpsio56ldo.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-92-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:34 2021]
rule discretize_counts:
    input: read_counts/AML-45-001_AD.csv, read_counts/AML-45-001_DP.csv
    output: input/AML-45-001_input.csv, input/AML-45-001.loci
    jobid: 45
    wildcards: patient=AML-45-001

[Fri Jan 29 23:13:35 2021]
Error in rule discretize_counts:
    jobid: 18
    output: input/AML-16-001_input.csv, input/AML-16-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp5qpx5amf.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-16-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:35 2021]
rule discretize_counts:
    input: read_counts/AML-96-001_AD.csv, read_counts/AML-96-001_DP.csv
    output: input/AML-96-001_input.csv, input/AML-96-001.loci
    jobid: 96
    wildcards: patient=AML-96-001

[Fri Jan 29 23:13:37 2021]
Error in rule discretize_counts:
    jobid: 50
    output: input/AML-50-001_input.csv, input/AML-50-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp57e5q_el.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-50-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:37 2021]
rule discretize_counts:
    input: read_counts/AML-60-001_AD.csv, read_counts/AML-60-001_DP.csv
    output: input/AML-60-001_input.csv, input/AML-60-001.loci
    jobid: 60
    wildcards: patient=AML-60-001

[Fri Jan 29 23:13:38 2021]
Error in rule discretize_counts:
    jobid: 22
    output: input/AML-20-001_input.csv, input/AML-20-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpbrfr6jw3.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-20-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:38 2021]
rule discretize_counts:
    input: read_counts/AML-95-001_AD.csv, read_counts/AML-95-001_DP.csv
    output: input/AML-95-001_input.csv, input/AML-95-001.loci
    jobid: 95
    wildcards: patient=AML-95-001

[Fri Jan 29 23:13:38 2021]
Error in rule discretize_counts:
    jobid: 45
    output: input/AML-45-001_input.csv, input/AML-45-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpl9p04e2e.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-45-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:38 2021]
rule discretize_counts:
    input: read_counts/AML-29-001_AD.csv, read_counts/AML-29-001_DP.csv
    output: input/AML-29-001_input.csv, input/AML-29-001.loci
    jobid: 31
    wildcards: patient=AML-29-001

[Fri Jan 29 23:13:38 2021]
Error in rule discretize_counts:
    jobid: 88
    output: input/AML-88-001_input.csv, input/AML-88-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpjdcan_qo.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-88-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:38 2021]
rule discretize_counts:
    input: read_counts/AML-69-001_AD.csv, read_counts/AML-69-001_DP.csv
    output: input/AML-69-001_input.csv, input/AML-69-001.loci
    jobid: 69
    wildcards: patient=AML-69-001

[Fri Jan 29 23:13:39 2021]
Error in rule discretize_counts:
    jobid: 60
    output: input/AML-60-001_input.csv, input/AML-60-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpc7fmgmfi.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:39 2021]
Error in rule discretize_counts:
    jobid: 96
[Fri Jan 29 23:13:39 2021]
    output: input/AML-96-001_input.csv, input/AML-96-001.loci

rule discretize_counts:
    input: read_counts/AML-21-001_AD.csv, read_counts/AML-21-001_DP.csv
    output: input/AML-21-001_input.csv, input/AML-21-001.loci
    jobid: 23
    wildcards: patient=AML-21-001
RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpv95rwjz6.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run

Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-96-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:39 2021]
rule discretize_counts:
    input: read_counts/AML-57-001_AD.csv, read_counts/AML-57-001_DP.csv
    output: input/AML-57-001_input.csv, input/AML-57-001.loci
    jobid: 57
    wildcards: patient=AML-57-001

[Fri Jan 29 23:13:42 2021]
Error in rule discretize_counts:
    jobid: 95
    output: input/AML-95-001_input.csv, input/AML-95-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpe883x16y.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-95-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:42 2021]
rule discretize_counts:
    input: read_counts/AML-99-001_AD.csv, read_counts/AML-99-001_DP.csv
    output: input/AML-99-001_input.csv, input/AML-99-001.loci
    jobid: 99
    wildcards: patient=AML-99-001

[Fri Jan 29 23:13:43 2021]
Error in rule discretize_counts:
    jobid: 31
    output: input/AML-29-001_input.csv, input/AML-29-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpjl9wwjk3.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-29-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:43 2021]
rule discretize_counts:
    input: read_counts/AML-94-001_AD.csv, read_counts/AML-94-001_DP.csv
    output: input/AML-94-001_input.csv, input/AML-94-001.loci
    jobid: 94
    wildcards: patient=AML-94-001

[Fri Jan 29 23:13:43 2021]
Error in rule discretize_counts:
    jobid: 69
    output: input/AML-69-001_input.csv, input/AML-69-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmphjmrjfwj.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-69-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:43 2021]
rule discretize_counts:
    input: read_counts/AML-36-001_AD.csv, read_counts/AML-36-001_DP.csv
    output: input/AML-36-001_input.csv, input/AML-36-001.loci
    jobid: 38
    wildcards: patient=AML-36-001

[Fri Jan 29 23:13:44 2021]
Error in rule discretize_counts:
    jobid: 57
    output: input/AML-57-001_input.csv, input/AML-57-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmps8_rzkpa.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-57-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:44 2021]
rule discretize_counts:
    input: read_counts/AML-49-001_AD.csv, read_counts/AML-49-001_DP.csv
    output: input/AML-49-001_input.csv, input/AML-49-001.loci
    jobid: 49
    wildcards: patient=AML-49-001

[Fri Jan 29 23:13:44 2021]
Error in rule discretize_counts:
    jobid: 23
    output: input/AML-21-001_input.csv, input/AML-21-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp06xxbzu5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-21-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:44 2021]
rule discretize_counts:
    input: read_counts/AML-58-001_AD.csv, read_counts/AML-58-001_DP.csv
    output: input/AML-58-001_input.csv, input/AML-58-001.loci
    jobid: 58
    wildcards: patient=AML-58-001

[Fri Jan 29 23:13:46 2021]
Error in rule discretize_counts:
    jobid: 49
    output: input/AML-49-001_input.csv, input/AML-49-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp9tl3r1dy.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:46 2021]
rule discretize_counts:
    input: read_counts/AML-71-001_AD.csv, read_counts/AML-71-001_DP.csv
    output: input/AML-71-001_input.csv, input/AML-71-001.loci
    jobid: 71
    wildcards: patient=AML-71-001

[Fri Jan 29 23:13:46 2021]
Error in rule discretize_counts:
    jobid: 99
    output: input/AML-99-001_input.csv, input/AML-99-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp78vky4dl.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-99-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:46 2021]
rule discretize_counts:
    input: read_counts/AML-53-001_AD.csv, read_counts/AML-53-001_DP.csv
    output: input/AML-53-001_input.csv, input/AML-53-001.loci
    jobid: 53
    wildcards: patient=AML-53-001

[Fri Jan 29 23:13:46 2021]
Error in rule discretize_counts:
    jobid: 94
    output: input/AML-94-001_input.csv, input/AML-94-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpwlwrnx4j.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-94-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:46 2021]
rule discretize_counts:
    input: read_counts/AML-30-001_AD.csv, read_counts/AML-30-001_DP.csv
    output: input/AML-30-001_input.csv, input/AML-30-001.loci
    jobid: 32
    wildcards: patient=AML-30-001

[Fri Jan 29 23:13:47 2021]
Error in rule discretize_counts:
    jobid: 38
    output: input/AML-36-001_input.csv, input/AML-36-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmphttzwovo.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-36-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:47 2021]
rule discretize_counts:
    input: read_counts/AML-18-001_AD.csv, read_counts/AML-18-001_DP.csv
    output: input/AML-18-001_input.csv, input/AML-18-001.loci
    jobid: 20
    wildcards: patient=AML-18-001

[Fri Jan 29 23:13:48 2021]
Error in rule discretize_counts:
    jobid: 58
    output: input/AML-58-001_input.csv, input/AML-58-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp0e6drxl_.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Removing output files of failed job discretize_counts since they might be corrupted:
input/AML-58-001_input.csv
Job failed, going on with independent jobs.

[Fri Jan 29 23:13:48 2021]
rule discretize_counts:
    input: read_counts/AML-25-001_AD.csv, read_counts/AML-25-001_DP.csv
    output: input/AML-25-001_input.csv, input/AML-25-001.loci
    jobid: 27
    wildcards: patient=AML-25-001

Terminating processes on user request, this might take some time.
[Fri Jan 29 23:13:48 2021]
Error in rule discretize_counts:
    jobid: 27
    output: input/AML-25-001_input.csv, input/AML-25-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpfors0k1g.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
[Fri Jan 29 23:13:48 2021]
Job failed, going on with independent jobs.
Error in rule discretize_counts:
    jobid: 20
    output: input/AML-18-001_input.csv, input/AML-18-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmp12r_lca5.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:13:48 2021]
Error in rule discretize_counts:
    jobid: 32
    output: input/AML-30-001_input.csv, input/AML-30-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpefkxzn6o.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:13:48 2021]
Error in rule discretize_counts:
    jobid: 71
    output: input/AML-71-001_input.csv, input/AML-71-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpodosd968.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
[Fri Jan 29 23:13:48 2021]
Error in rule discretize_counts:
    jobid: 53
    output: input/AML-53-001_input.csv, input/AML-53-001.loci

RuleException:
CalledProcessError in line 26 of /scratch/data/leah/phyolin_private/aml_patients/Snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /scratch/data/leah/phyolin_private/aml_patients/.snakemake/scripts/tmpzdiz6qll.discretize_counts.R' returned non-zero exit status 1.
  File "/scratch/data/leah/phyolin_private/aml_patients/Snakefile", line 26, in __rule_discretize_counts
  File "/scratch/data/leah/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.
Complete log: /scratch/data/leah/phyolin_private/aml_patients/.snakemake/log/2021-01-29T231301.703890.snakemake.log
